_data/community-stories.json

```diff
@@ -1306,5 +1306,283 @@
+      "created_at": "2021-06-04T00:22:05.644Z"
+    },
+    {
+      "name": "Jennifer K.",
+      "email": "Pengravel@gmail.com",
+      "site": "Mason",
+      "picture": "https://images.ctfassets.net/y3a9myzsdjan/2EKgqqGNPgWqbHKtLq1Ox2/24444aaf57ac902d86c92c4362a5a275/Screenshot_20210321-204750_Chrome.jpg",
+      "message": "During a recent women's Bible study at Crossroads in spring 2021, I made the decision to be baptized and make public my journey in saying yes to seeking and following a perfect Jesus.",
+      "yes": "on",
+      "ip": "50.5.26.40",
+      "user_agent": "Mozilla/5.0 (Linux; Android 9; SM-G950U) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.210 Mobile Safari/537.36",
+      "referrer": "https://www.crossroads.net/my25/?utm_medium=email&_hsmi=131427033&_hsenc=p2ANqtz-_uhJCxpDiF3A7-xgzoHZmtSK-C9mCraVZTyQLr8Tegbuwr6J4YVxU8AJQcYLg8muY-OlnC7WQC9Ixzlj6RR5EjMrQEsg&utm_content=131427033&utm_source=hs_email",
+      "created_at": "2021-06-04T00:01:13.066Z"
+    },
+    {
+      "name": "Tiffany B.",
+      "email": "Tiffanybrearton@gmail.com",
+      "site": "Florence",
+      "picture": "https://images.ctfassets.net/y3a9myzsdjan/7IpgK4saoVwEZWdG1F9EUQ/54df13f510a25039e2d9106d837bcca0/inbound3720310051756582272.jpg",
+      "message": "When I started Crossroads the past of my sexual abuse could no longer be pushed down and I was desperate for healing. I heard about woman camp and left with a new sense of freedom. I joined at group at church called CST and found so much healing. God has transformed me and Crossroads has helped.",
+      "yes": "on",
+      "ip": "2600:387:f:4a1a::2",
+      "user_agent": "Mozilla/5.0 (Linux; Android 11; SM-G996U Build/RP1A.200720.012; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/90.0.4430.210 Mobile Safari/537.36 [FB_IAB/FB4A;FBAV/318.0.0.39.154;]",
+      "referrer": "https://www.crossroads.net/my25/",
+      "created_at": "2021-06-03T00:36:44.281Z"
+    },
+    {
+      "name": "Ellen K.",
+      "email": "ellen.kiser@crossroads.net",
+      "site": "Lexington",
+      "picture": "https://images.ctfassets.net/y3a9myzsdjan/2vWWWphNRdTQ23d5Yq4iaj/6dd1e85ef7635838fe9016cbf522318e/my25.jpg",
+      "message": "Crossroads is where I first learned what real community looks like. I've had leaders who really care about me. I've made friends who encourage me. I have coworkers who believe in me. Those people have helped me feel less insecure & more confident in myself. I know that I'm not alone.",
+      "yes": "on",
+      "ip": "70.62.166.50",
+      "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36",
+      "referrer": "https://www.crossroads.net/my25/",
+      "created_at": "2021-06-02T17:59:56.774Z"
+    }
+  ]
```

Comment:
Please Remove this `]`.

The overall structure should be.

```json
[
  {},
  {},
  {},
  {}
]
```

what you have is.
```json
[
  {},
  {},
  [
      {},
      {}
  ]
]
```